src/main/java/seedu/address/model/pilot/Pilot.java

```diff
@@ -97,6 +98,18 @@ public int getFlightHour() {
         return flightHour;
     }
 
+    @Override
+    public List<String> getDisplayList() {
+        return List.of(
+            "UUID: " + id,
+            "Name: " + name,
+            "Gender: " + gender,
+            "Age: " + age,
+            "Rank: " + rank,
+            "Flight Hour: " + flightHour
+        );
+    }
+
```

Comment:
professional indentation, good

btw, why not use String.format?

Reply:
I agree that String.format would be better, as in #53